Userland/Libraries/LibWeb/HTML/HTMLElement.cpp

```diff
@@ -645,6 +645,34 @@ WebIDL::ExceptionOr<JS::NonnullGCPtr<ElementInternals>> HTMLElement::attach_inte
     return { internals };
 }
 
+// https://html.spec.whatwg.org/multipage/popover.html#dom-popover
+Optional<StringView> HTMLElement::popover() const
```

Comment:
Let's return `Optional<String>` here.

- If we have the attribute, great, we return an existing string, no allocation!
- If we don't have the attribute, but fall back to "auto" or "manual", great, both are <=7 ASCII characters and can use the "short string optimization" in `String` -- no allocation!